library.js

```diff
@@ -137,6 +149,11 @@ function doFindOrCreateUser(user, callback) {
 
   pino.info({ method: 'doFindOrCreateUser', input: user, type: 'start' });
 
+  if (!user.email || (user.email && user.email.lenght < 5)) {
+    return callback(new Error("ERROR-EMAIL"), null);
```

Comment:
Just a naming convention. Lets use `_` while using capital case values. Ex: `ERROR_EMAIL` or perhaps, `INVALID_EMAIL`. As a developer, we want to make the next developer, understand our code as easy as possible. That is the main reason for following best practices & conventions.